makefile-docker.defs

```diff
@@ -113,14 +113,25 @@ test_search:
 		-vvv tests/search \
 		-m 'search' ${ARGS}
 
-test_cors: 
+.ONESHELL:
+test_use_cases:
+	docker compose \
+		-f docker-compose.yml \
+		-f docker-compose.dev.yml \
+		run --rm --name search_test \
+		-v "${PWD}/data:/data" \
+		backend pytest \
+		-vvv tests/use-cases \
+		-m 'use_case' ${ARGS}
+
+test_cors:
 	docker compose -f docker-compose.yml -f docker-compose.dev.yml run --rm backend pytest -vvv -m 'cors' ${ARGS}
 
 test_unit:
 	docker compose -f docker-compose.yml -f docker-compose.dev.yml run --rm backend pytest -vvv tests/unit ${ARGS}
 
 test_non_search:
-	docker compose -f docker-compose.yml -f docker-compose.dev.yml run --rm backend pytest -vvv -m 'not search' ${ARGS}
+	docker compose -f docker-compose.yml -f docker-compose.dev.yml run --rm backend pytest -vvv -m 'not search and not use_case' ${ARGS}
```

Comment:
This is not really scalable - we should perhaps mark all old "non-search" tests as such to distinguish them ?